### PR TITLE
Fixed Display Board, added game logic tests for variable sizes

### PR DESCRIPTION
**Pull request by skp96**
Note: this is not being merged into main

### CODE REVIEW

File: core_tic_tac_toe/display.py [hunk 1]
```diff
@@ -48,19 +46,30 @@ def clear_console(self):
...
+
+        return vertical_bars
 
     def __generate_rows(self, move):
         row = ""
 
         for idx, position in enumerate(move):
             if idx != len(move) - 1:
-                row += "  {}  |".format(position)
+                if type(position) == int and position // 10 > 0:
```
Comment (cdemyanovich) [inline core_tic_tac_toe/display.py]: Why `10`?

Comment (skp96) [reply core_tic_tac_toe/display.py]: The format for double digit numbers is different than the format for single digit numbers. I'll go ahead and extract that piece of logic into its own method so I can give it a more descriptive name.

File: core_tic_tac_toe/display.py [hunk 2]
```diff
@@ -65,11 +65,14 @@ def __generate_rows(self, move):
 
         for idx, position in enumerate(move):
             if idx != len(move) - 1:
-                if type(position) == int and position // 10 > 0:
+                if type(position) == int and self.__is_double_digit(position):
```
Comment (cdemyanovich) [inline core_tic_tac_toe/display.py]: If you always want to allow a certain amount of space, you can probably use some additional notation from [Python's format string syntax](https://docs.python.org/3/library/string.html#format-string-syntax).

Comment (skp96) [reply core_tic_tac_toe/display.py]: Thank you for sending this over! I was able to find the correct formatting type.

**Review by cdemyanovich — APPROVED** [review]

**Pull request merged**